### ISSUE TITLE
Insufficient funds

### ISSUE THREAD

**Issue by alar0n** [iron-fish/ironfish#2129]
### What happened?

everything can be seen in the screenshot. Sufficient funds, but not enough

![image](https://user-images.githubusercontent.com/31000576/187069741-339a6622-c79c-4260-b990-272e0b389abb.png)

### Version

 0.1.44

### Debug output

```shell
yarn run v1.22.19
$ yarn build && yarn start:js debug
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node --expose-gc bin/run debug
Database in use, skipping output that requires database.
Stop the node and run the debug command again to show full output.

Iron Fish version               0.1.44 @ src
Iron Fish library               0.0.21 @ src
Operating system                Linux x64
CPU model(s)                    Intel(R) Xeon(R) Gold 6248R CPU @ 3.00GHz
CPU threads                     40
RAM total                       31.38 GiB
Heap total                      4.01 GiB
Node version                    v16.16.0
ironfish in PATH                true
Garbage Collector Exposed       true
Telemetry enabled               true
Node name                       alar0n
Block graffiti                  alar0n
```

### Graffiti

alar0n

**Comment by alar0n** [comment]
Problem still exist in 0.1.45:
![image](https://user-images.githubusercontent.com/31000576/187340940-844b714d-b930-4423-b0cd-ac5f7b93206f.png)

**Comment by hughy** [comment]
The Wallet 2.0 refactor released in [v0.1.48](https://github.com/iron-fish/ironfish/releases/tag/v0.1.48) included significant changes to the way that account balances are stored and calculated.

Please open a new issue if you run into this again!
